src/main/web/components/forms/inputs/DropdownFormSelector.tsx

```diff
@@ -0,0 +1,337 @@
+/**
```

Comment:
this component is mostly a copy-and-paste of `SelectInput`.

The main difference is that you can pass more than 1 nested form to it and user can choice which form wants to open.